resources/puppetlabs/lein-ezbake/template/global/ext/cli/reload.erb

```diff
@@ -13,6 +14,7 @@ if [ -r "$restartfile" ];  then
         echo "Service not running so cannot be reloaded" 1>&2
         exit 1
     fi
+    sleep 0.1
```

Comment:
It's arbitrary.  It makes the assumption that it may take the application a little bit of time - but not much -- to process the HUP signal and reload in cases where there's no work in process that has to be gracefully stopped.  Just seemed like for cases where the reload is fairly quick that we'd have a better chance of being able to detect it has finished here before going into the 1 second polling loop.  I took this idea from the killproc implementation from the init function in some OS distributions about how they wait for a process to stop - quick sleep up front (.1 second) followed by the 1 second sleeps later if it doesn't complete right away.

All that said, I'm good with removing this line.  I don't care all that much.